src/main/java/com/alkemy/ong/service/impl/TestimonialServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.alkemy.ong.service.impl;
+
+import com.alkemy.ong.dto.TestimonialDto;
+import com.alkemy.ong.model.Testimonials;
+import com.alkemy.ong.repository.TestimonialsRepository;
+import com.alkemy.ong.service.ITestimonialService;
+import com.alkemy.ong.util.MessageHandler;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class TestimonialServiceImpl implements ITestimonialService {
+
+    @Autowired
+    private TestimonialsRepository testimonialsRepository;
+
+    @Autowired
+    private MessageHandler messageHandler;
+
+    @Autowired
+    private ModelMapper modelMapper;
+
+
+    @Override
+    public List<TestimonialDto> findAll() {
```

Comment:
tanto a este como a los metodos que retornan null (que son los otros metodos CRUD que agregaste al service pero no estaban en tu ticket) agregale una linea que diga `//TODO` o `//TODO Not implemented yet`. Es algo que se suele agregar para remarcar que hay codigo por escribir, el IDE los identifica de otra manera, como palabra reservada y luego se puede filtrar, otra anotacion similar es la de //FIXME